share/poly_test.go

```diff
@@ -317,3 +318,132 @@ func TestPriPolyCoefficients(test *testing.T) {
 	require.Equal(test, a.coeffs, b.coeffs)
 
 }
+
+func TestRefreshDKG(test *testing.T) {
```

Comment:
s/test/t/ to keep with go standards, please.

Reply:
We are using `t` as the threshold variable name which is the standard notation in the literature. To avoid conflicts we use `test` as the parameter here. That okay?

Reply:
Oh yeah, that makes sense. Forgot about that particularity of these files.